rclpy/rclpy/__init__.py

```diff
@@ -16,6 +16,7 @@
 
 from rclpy.executors import SingleThreadedExecutor as _SingleThreadedExecutor
 from rclpy.impl.implementation_singleton import rclpy_implementation as _rclpy
+import rclpy.logging  # noqa
```

Comment:
Why import this in `__init__,py`? Is it to initialize the root_logger before `rclpy.init()` is called? What breaks if this import doesn't happen here?

Reply:
nothing breaks, it's just to save users from having to import it themselves in each script that wants to use logging. I can add a comment for it but I'm not sure which other lines in this section are doing it for the same reason

Reply:
I don't fully understand `__init__.py`, but if I comment this line out it appears to be imported regardless.

```
$ python3
Python 3.5.2 (default, Nov 17 2016, 17:05:23) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import rclpy
>>> rclpy.logging.logerr("Hello World!")
[ERROR] []: Hello World! (<module>() at /workspace/ros2_ws/<stdin>:1)
True
```

Reply:
ah, it was necessary on master to use the non-node logging methods, but since node.py is importing it now, it's not needed here: a48d6b634c08751d6af302b8c4706c861a33df9a